CMakeLists.txt

```diff
@@ -69,9 +69,6 @@ include(${_ALPAKA_ADD_EXECUTABLE_FILE})
 set(_ALPAKA_ADD_LIBRARY_FILE "${_ALPAKA_ROOT_DIR}/cmake/addLibrary.cmake")
 include(${_ALPAKA_ADD_LIBRARY_FILE})
 
-# Add module search path
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${_ALPAKA_ROOT_DIR}/cmake/modules/")
-
 # parse environment variable `CMAKE_MODULE_PATH`
 if(DEFINED ENV{CMAKE_MODULE_PATH})
     string(REPLACE ":" ";" ENV_CMAKE_MODULE_PATH $ENV{CMAKE_MODULE_PATH})
```

Comment:
I don't know if we still need this.

Reply:
ROCm comes with its own `FindHIP.cmake` nowadays and a user should point to its location on the command line. IMO this can be removed, too.

Reply:
So users will never define an environment variable `CMAKE_MODULE_PATH` to inject `FindHIP.cmake`?

Reply:
Ok, removed.

Reply:
To expand a bit on this: The user should provide the correct path by setting one of the following:

1. The `<PackageName>_ROOT` environment variable / CMake variable.
2. The `CMAKE_PREFIX_PATH` CMake variable.
3. The `<PackageName>_DIR` environment variable.
4. The user package registry - [Link](https://cmake.org/cmake/help/v3.18/manual/cmake-packages.7.html#user-package-registry).

The top overrides the bottom, i.e. `<PackageName>_ROOT` will override anything else, `CMAKE_PREFIX_PATH` will override `<PackageName>_DIR` and the package registry, and so on.

Reply:
thx for the explanation!